### DISCUSSION TITLE
I Add as 2FA a mobile App. But I see the 2FA-Warning banner if I locked In. What should I do?

### DISCUSSION THREAD

**Discussion by wheelseal** [community/community#129876] [Other Feature Feedback, Questions, & Ideas]
Hi at all,
today i locked in to Github I get the Warning "Please configure another 2FA method to reduce your risk of permanent account lockout. We strongly recommend against SMS as it is prone to fraud and delivery may be unreliable depending on your region."
I Already set a Mobile App for 2FA but the banner not disappears. I changed that from SMS to App for a long time ago. What should I do?

**Comment by davevad93** [comment]
Hi @wheelseal ,

The warning suggests to enable more than one 2FA method and avoid SMS authentication, you still see the warning because you only have one 2FA method enabled, if you enable one more (even SMS) the warning disappears.

For more reference see these discussions: #129189 , #129638
